pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -57,10 +58,12 @@ func defaultPredicates() sets.String {
 }
 
 // ApplyFeatureGates applies algorithm by feature gates.
-func ApplyFeatureGates() {
+func ApplyFeatureGates() []api.FeatureDependency {
```

Comment:
nit: Update the comment here as this function's signature changed.

Reply:
done